frontend/app/src/components/import/ImportSource.vue

```diff
@@ -126,15 +130,35 @@ export default defineComponent({
       return displayDateFormatter.format(now, get(dateInputFormat)!);
     });
 
+    const taskType = TaskType.IMPORT_CSV;
+    const { awaitTask, isTaskRunning } = useTasks();
+
+    const loading = isTaskRunning(taskType);
+
     const uploadPackaged = async (file: string) => {
       try {
-        await api.importDataFrom(
+        const { taskId } = await api.importDataFrom(
```

Comment:
@lukicenturi any idea what happens if you navigate away while the task is pending?

Will the task still get removed from the notifications?

Reply:
Yeah it is still get removed